spec/pubid_iso/identifier_spec.rb

```diff
@@ -762,6 +762,14 @@ module Pubid::Iso
         end
       end
 
+      context "when create document with amendment without year" do
+        let(:params) { { amendments: [Pubid::Iso::Amendment.new(number: 1)] } }
+
+        it "renders document with amendment year" do
+          expect(subject.to_s).to eq("ISO 123/Amd 1")
```

Comment:
As previously mentioned in the other ticket, "ISO 123/Amd 1" cannot exist. The base document must have a year.

i.e. "ISO 123:1999/Amd 1" is okay, but not "ISO 123/Amd 1".